dashbuilder-client/dashbuilder-displayer-client/src/main/java/org/dashbuilder/displayer/client/widgets/group/ColumnFunctionEditor.java

```diff
@@ -108,6 +109,7 @@ public void init(DataSetMetadata metadata,
         this.groupFunction = groupFunction;
         this.targetType = targetType;
         this.metadata = metadata;
+        this.functionsEnabled = functionsEnabled;
 
         columnDetailsEditor.init(metadata, this.groupFunction);
```

Comment:
IntelliJ static analysis discovered an initialization issue in [related code](https://github.com/dashbuilder/dashbuilder/blob/master/dashbuilder-client/dashbuilder-displayer-client/src/main/java/org/dashbuilder/displayer/client/widgets/group/ColumnDetailsEditorView.java#L46)
Instead of 
`this.presenter = presenter`
is should be 
`this.presenter = columnDetailsEditor`, otherwise the presenter remains uninitialized